newdle/migrations/versions/20200914_1640_aa8c14c9e867_.py

```diff
@@ -0,0 +1,28 @@
+"""Add creator email column to db model
```

Comment:
Both this and the other file lack the title slug in the filename (usually it's set when doing `flask db migrate -m 'title here'`).